lib/lost-align.js

```diff
@@ -26,6 +26,11 @@ module.exports = function lostAlignDecl(css, settings) {
   css.walkDecls('lost-align', function(decl) {
     var declArr = [];
     var lostAlign;
+    var cloneAllBefore = function(props) {
+      for (var prop in props) {
+        decl.cloneBefore({prop: prop, value: props[prop]})
+      }
+    };
```

Comment:
This is what I was thinking. I'd really like to double check with PostCSS to make sure there's not already something like this that they've introduced to the API (it's been awhile since I've looked over all the PostCSS API).

If not, it'd be nice if this could get merged into PostCSS as it's super handy.

If not, then yeah, let's put this everywhere in Lost.